### PR TITLE
Addition of License File

### PR DESCRIPTION
**Pull request by fmalatino**
**Description**
Added Apache 2.0 license text 

Fixes # 22
This PR addresses 22

**How Has This Been Tested?**
No changes to functionality of code

**Checklist:**
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] New check tests, if applicable, are included
- [ ] Targeted model, if this change was triggered by a model need/shortcoming

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2022 Allen Institute for Artificial Intelligence (AI2)
```
Comment (FlorianDeconinck) [inline LICENSE.md]: This needs to go

Comment (fmalatino) [reply LICENSE.md]: Just the copyright and reference to AI2? Or more?

Comment (FlorianDeconinck) [reply LICENSE.md]: The copyright yeah

**Review by FlorianDeconinck — DISMISSED** [review]
LGTM

**Review by bensonr — DISMISSED** [review]

File: setup.py [hunk 2]
```diff
@@ -36,7 +36,7 @@
     classifiers=[
```
Comment (oelbert) [inline setup.py:36]: We should change this

**Review by oelbert — CHANGES_REQUESTED** [review]
I think we should update the author list and contact info as well if we're updating the licensing

File: setup.py (hunk 2 above)
Comment (fmalatino) [reply setup.py:36]: Is there something it should be?

Comment (oelbert) [reply setup.py:36]: Ugh it didn't highlight the whole portion I wanted. Basically replace 
```
setup(
     author="Vulcan Technologies LLC",
    author_email="elynnw@vulcan.com",
    python_requires=">=3.11.7",
    classifiers=[
```
with 
```
setup(
    author="NOAA - Geophysical Fluid Dynamics Laboratory",
    author_email="oliver.elbert@noaa.gov",
    python_requires=">=3.11.7",
    classifiers=[
```

Comment (oelbert) [reply setup.py:36]: Or anyone else who wants to be Point of Contact ;-)

**Review by oelbert — APPROVED** [review]
👍

**Pull request merged**